WordPress/src/main/res/layout/my_site_fragment.xml

```diff
@@ -2,634 +2,677 @@
+                    android:layout_height="36dp"
+                    android:layout_gravity="center"
+                    android:layout_marginBottom="@dimen/margin_extra_extra_medium_large"
+                    android:layout_marginTop="@dimen/margin_medium"
+                    android:background="?attr/selectableItemBackgroundBorderless"
+                    android:ellipsize="end"
+                    android:fontFamily="sans-serif-medium"
+                    android:gravity="center_vertical"
+                    android:maxLines="1"
+                    android:text="@string/my_site_btn_switch_site"
+                    android:textAllCaps="true"
+                    android:textColor="@color/blue_50"
+                    android:textSize="@dimen/text_sz_medium"/>
+
+                <LinearLayout
+                    android:id="@+id/quick_action_buttons_container"
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:layout_marginBottom="@dimen/margin_my_site_quick_actions_bottom"
+                    android:orientation="horizontal"
+                    android:weightSum="100">
+
+                    <LinearLayout
+                        style="@style/MySiteQuickActionButtonContainer">
+
+                        <com.google.android.material.floatingactionbutton.FloatingActionButton
+                            android:id="@+id/quick_action_stats_button"
+                            style="@style/MySiteQuickActionButton"
+                            android:src="@drawable/ic_stats_alt_white_24dp"/>
 
                         <org.wordpress.android.widgets.WPTextView
-                            android:layout_height="match_parent"
-                            android:layout_width="wrap_content"
-                            android:text="@string/my_site_btn_switch_site"
-                            android:textAllCaps="true"
-                            android:textColor="@color/neutral_70"
-                            android:textSize="@dimen/text_sz_small"
-                            style="@style/MySiteListRowTextView" >
-                        </org.wordpress.android.widgets.WPTextView>
+                            style="@style/MySiteQuickActionButtonLabel"
+                            android:text="@string/stats"/>
 
                     </LinearLayout>
 
-                </RelativeLayout>
+                    <LinearLayout
+                        android:id="@+id/quick_action_pages_container"
+                        style="@style/MySiteQuickActionButtonContainer">
 
-            </androidx.cardview.widget.CardView>
+                        <com.google.android.material.floatingactionbutton.FloatingActionButton
+                            android:id="@+id/quick_action_pages_button"
+                            style="@style/MySiteQuickActionButton"
+                            android:backgroundTint="@android:color/white"
```

Comment:
```suggestion
```

We can remove the `android:backgroundTint="@android:color/white"` attribute from the `quick_action_pages_button` view since the `MySiteQuickActionButton` style already has that attribute.  That would eliminate a redundancy and make the view consistent with the other Quick Action buttons.